src/constants.py

```diff
@@ -25,8 +25,14 @@
 # cap parameter, per CIP-20, measured in ETH
 CAP_PARAMETER = 0.01
 
-# requests
+# threshold parameter to generate an alert when receiving kickbacks
```

Comment:
Should we move some of the more test specific constants into the test itself?

Reply:
Don't have a strong opinion on that. I could do that if you think the constants file has grown a lot.

Reply:
Then lets keep as is and refactor later if necessary.